manim/utils/color.py

```diff
@@ -84,11 +84,13 @@
     "ORANGE",
 ]
 
-from enum import Enum
 import random
+import re
```

Comment:
looks like an unused import...

Reply:
I use it in hex_to_rgb to enforce sensible behaviour for hex strings